corehq/apps/cloudcare/static/cloudcare/js/backbone/shared.js

```diff
@@ -24,7 +24,7 @@ var Selectable = Backbone.View.extend({
         }
         if (this.selected) {
```

Comment:
i'm thinking this could benefit from a slight refactor since the logic is getting a bit hard to follow. we can substantially cut down the lines here by pulling the confirmation and dirty check above the `if (this.selected)`